README.md

```diff
@@ -69,6 +72,23 @@ Time: 00:00.962, Memory: 8.00 MB
 OK (1 test, 1 assertion)
 ```
 
+For identityToken generated from _refresh_token_;
+If you don't want to copy-paste above code you can paste freshly generated `identityToken` in `tests/E2eRefreshToken/Auth/AppleJwtFetchingServiceTest.php:53`
```

Comment:
Please move this test from `E2eRefreshToken/Auth` to `E2e/Auth` directory. You can keep the test in a separate file (require renaming) or add it to the existing one.